volto/intro.rst

```diff
@@ -0,0 +1,44 @@
+.. _volto_intro-label:
+
+============
+Introduction
+============
+
+Who are you?
+============
+
+Tell us about yourselves:
+
+* Name, company, country...
+* What is your Plone experience?
+* What is your JavaScript experience?
+* What are your expectations for this tutorial?
+
+.. _intro-what-will-we-do-label:
+
+What will we do?
+================
+
+Some technologies and tools we use during the training:
+
+* React https://reactjs.org/
+* create-react-app https://github.com/facebook/create-react-app
+* Yarn https://yarnpkg.com
+* JSX
+* Redux https://redux.js.org
+* React-Router https://reacttraining.com/react-router/
+* Volto https://github.com/plone/volto
+
+.. _intro-what-to-expect-label:
+
+What to expect
+==============
+
+At the end of the course you'll know how to create a custom website using Volto.
```

Comment:
Avoid contractions, because they don't translate easily. Google Translate gets most contractions correctly, but we shouldn't leave it up to a third party.

s/you'll/you will